src/VirtoCommerce.AuthorizeNet.Web/module.manifest

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<module>
+    <id>VirtoCommerce.AuthorizeNet</id>
+    <version>3.0.0</version>
+	  <version-tag>rc.3</version-tag>
+    <platformVersion>3.0.0</platformVersion>
+    <dependencies>
+      <dependency id="VirtoCommerce.Store" version="3.0.0" />
+			<dependency id="VirtoCommerce.Orders" version="3.0.0-rc.3-001" />
```

Comment:
Use version="3.0.0" for all dependencies

Reply:
Done